spec/factories/customer_factory.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :customer do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name}
+
+  end
+end
```

Comment:
This is a nice setup and I can reference this code when trying to use factorybot